core/base/document.php

```diff
@@ -540,6 +540,8 @@ protected function get_initial_config() {
 			],
 		];
 
+		do_action( 'elementor/document/get_config', $this );
```

Comment:
Move up before the initiation of `$config`
```suggestion
		do_action( 'elementor/document/before_get_config', $this );
```